src/__stories__/Theme.stories.mdx

```diff
@@ -0,0 +1,293 @@
+import { Meta } from '@storybook/addon-docs'
+import BorderedBox from '../components/BorderedBox'
+import Typography from '../components/Typography'
+import * as localContrasts from '../theme/colors/contrasts'
+import theme from '../../src/theme'
+import { Global, ThemeProvider } from '@emotion/react'
+import Separator from '../components/Separator'
+import { globalStyles } from '../../.storybook/preview'
+import { readableColor } from 'polished'
+import { Row } from '../components'
+import Alert from '../components/Alert'
+
+<Meta title="Theme" />
+
+# Theming
+
+Scaleway UI is based on `@emotion/react` library. It allows you to customize the theme as you need.
+
+## Colors
+
+Scaleway UI provide you a set of colors trough theme, those colors are defined by tokens following asana token pattern.
+
+Tokens are designed as follows: `sentiment.usageProminenceInteraction`
+
+Where:
+
+- **sentiment** = `primary`, `secondary`, `neutral`, `info`, `danger`, `warning` and `success`
+- **usage** = `text`, `background` and `border`.
+- **prominence** = `weak` and `strong`.
+- **interaction**: `disabled` and `hover`.
+
+<p>&nbsp;</p>
```

Comment:
🤔

Reply:
I think this is intended as multiple carriage return as mdx squash multiple lines into only one